packages/expo-store-review/src/ExpoStoreReview.ts

```diff
@@ -1,2 +1,13 @@
-import { NativeModulesProxy } from '@unimodules/core';
-export default NativeModulesProxy.ExpoStoreReview;
+import { Platform } from '@unimodules/core';
+
+export default {
+  get name(): string {
+    return 'ExpoStoreReview';
+  },
+  async isAvailableAsync(): Promise<boolean> {
+    // true on Android, false on web
+    return Platform.OS !== 'web';
+  },
+  // Unimplemented on web and Android
+  requestReview: null as null | (() => Promise<void>),
+};
```

Comment:
The old way of organizing things makes more sense IMO: most platforms (except web) can potentially define `NativeModulesProxy.ExpoStoreReview` and that is the default. For web we have the special web.ts file and since Android is currently an exception and doesn't define `NativeModulesProxy.ExpoStoreReview` we have the android.ts file.